style.css

```diff
@@ -34,11 +32,15 @@
   flex: none;
   order: 0;
   flex-grow: 0;
+  margin-left: 15px;
+}
+
+.nav-menu {
+  display: none;
 }
```

Comment:
- [ ] As the previous reviewer suggested, kindly try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=iqWpkNWp3niX7g3e-0). Therefore I suggest that you ensure the color of the nav links match that of Figma design 😎 

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/221373501-c703d724-51e5-4055-a242-663cbf3cfbfa.png)